src/esp/scene/SceneNode.cpp

```diff
@@ -16,13 +17,17 @@ SceneNode::SceneNode(SceneNode& parent)
   setId(parent.getId());
   setCachedTransformations(Mn::SceneGraph::CachedTransformation::Absolute);
   absoluteTransformation_ = absoluteTransformation();
+  nodeSensorSuite_ = new esp::sensor::SensorSuite(*this);
```

Comment:
Don't these need corresponding destructors?

Reply:
@Skylion007 : No, they do not.
`new esp::sensor::SensorSuite(*this)` will create this feature in memory, and attach it to the node (`*this`), which means the node owns this feature (`sensorSuite`). When the node gets deleted, it will first release the attached features. 
the raw pointer `nodeSensorSuite_` is to provide a shortcut to locate this feature, so that no need to traverse the entire feature list of the node.

Reply:
@vivian: Would you consider putting the above reply into the comments so people will not be confused in the future? thanks.